packages/admin-ui-extensions-react/src/components/Button/index.ts

```diff
@@ -1,2 +1,5 @@
+import type {ReactPropsFromRemoteComponentType} from '@remote-ui/react';
+import type {ButtonProps as BaseButtonProps} from '@shopify/admin-ui-extensions';
+
+export type ButtonProps = ReactPropsFromRemoteComponentType<BaseButtonProps>;
```

Comment:
Admin UI Extension Button doesn't have children. So, `BaseButtonProps` is enough.

Reply:
Oops thanks